ote_sdk/ote_sdk/tests/entities/test_annotation.py

```diff
@@ -0,0 +1,607 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+import datetime
+from typing import List
+
+import pytest
+
+from ote_sdk.entities.annotation import (
+    Annotation,
+    AnnotationSceneEntity,
+    AnnotationSceneKind,
+    NullAnnotationSceneEntity,
+)
+from ote_sdk.entities.color import Color
+from ote_sdk.entities.id import ID
+from ote_sdk.entities.label import Domain, LabelEntity
+from ote_sdk.entities.scored_label import ScoredLabel
+from ote_sdk.entities.shapes.ellipse import Ellipse
+from ote_sdk.entities.shapes.polygon import Point, Polygon
+from ote_sdk.entities.shapes.rectangle import Rectangle
+from ote_sdk.tests.constants.ote_sdk_components import OteSdkComponent
+from ote_sdk.tests.constants.requirements import Requirements
+
+
+@pytest.mark.components(OteSdkComponent.OTE_SDK)
+class TestAnnotation:
+
+    rectangle = Rectangle(x1=0.5, x2=1.0, y1=0.0, y2=0.5)
+    labels = []  # type: List[ScoredLabel]
+    annotation = Annotation(shape=rectangle, labels=labels)
+
+    car = LabelEntity(
+        id=123456789,  # type: ignore
```

Comment:
What do you ignore here, why not to use `id=ID(123456789)`

Reply:
Fixed